app/scenes/channel/channel.js

```diff
@@ -77,7 +83,10 @@ export default class Channel extends React.Component {
         }
 
         return (
-            <View style={{flex: 1, backgroundColor: theme.centerChannelBg}}>
+            <KeyboardAvoidingView
```

Comment:
It works as expected this way. The `ChannelHeader` remains at the top, the `PostTextbox` at the bottom, and the `ChannelPostList` shrunk to still fit between them.

And dismissing the keyboard is done automatically when the focus leaves the textbox